frontend/src/Calendar/Agenda/Agenda.js

```diff
@@ -32,7 +68,9 @@ function Agenda(props) {
 }
 
 Agenda.propTypes = {
-  items: PropTypes.arrayOf(PropTypes.object).isRequired
+  items: PropTypes.arrayOf(PropTypes.object).isRequired,
+  start: PropTypes.date,
+  end: PropTypes.date
```

Comment:
These (start, end) should be `PropTypes.string.isRequired`... date isn't a valid value and gives warning:

```
checkPropTypes.js:20 Warning: Failed prop type: Agenda: prop type `end` is invalid; it must be a function, usually from the `prop-types` package, but received `undefined`.
    in Agenda (created by ConnectFunction)
    in ConnectFunction (at Calendar.js:43)
    in div (at Calendar.js:41)
    in div (at Calendar.js:26)
    in Calendar (at CalendarConnector.js:165)
    in CalendarConnector (created by ConnectFunction)
```